src/Components/Featured/index.js

```diff
@@ -8,10 +8,10 @@ export default class Featured extends Component {
         <iframe
           width="100%"
           height="100%"
-          src={"https://www.youtube.com/embed/" + this.props.data[1].youtube}
-          frameborder="0"
+          src={`https://www.youtube.com/embed/${this.props.data[1].youtube}?autoplay=1`}
+          frameBorder="0"
```

Comment:
Can we wait on putting the autoplay till the end. That way we don't have to pause the video every time while we're troubleshooting.